static/styles/zulip.css

```diff
@@ -1797,16 +1796,13 @@ nav a.no-style:hover {
     cursor: pointer;
 }
 
-nav .column-left .nav-logo {
+nav .column-left img {
```

Comment:
Mmm, this should still be `.nav-logo`, otherwise, the entire block is applied to all `<img>` instead of just to the `.nav-logo`.